espresso-server/app/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ dependencies {
     androidTestImplementation("androidx.test:rules:${Version.testlib}")
     androidTestImplementation("com.google.code.gson:gson:${Version.gson}")
     androidTestImplementation("org.nanohttpd:nanohttpd-webserver:${Version.nanohttpd}")
-    androidTestImplementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Version.kotlin}")
+    androidTestImplementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Version.kotlin}")
```

Comment:
Tried to use `implementation`, with JDK 11, with kotlin 1.4.31 etc, but still no luck.
```
[debug] [Espresso] [Instrumentation] io.appium.espressoserver.EspressoServerRunnerTest:
[debug] [Espresso] [Instrumentation] Error in startEspressoServer(io.appium.espressoserver.EspressoServerRunnerTest):
[debug] [Espresso] java.lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/internal/Intrinsics;
[debug] [Espresso] 	at io.appium.espressoserver.lib.helpers.AndroidLogger.info(Unknown Source:2)
[debug] [Espresso] 	at io.appium.espressoserver.lib.http.Server.stop(Server.kt:103)
[debug] [Espresso] 	at io.appium.espressoserver.EspressoServerRunnerTest.startEspressoServer(EspressoServerRunnerTest.kt:51)
[debug] [Espresso] 	at java.lang.reflect.Method.invoke(Native Method)
```

Reply:
this issue has been resolved